etc/ui-config/mh_default_org/paella7/config.json

```
@@ -69,7 +69,8 @@
             "presenter/player+preview",
             "presentation/player+preview"
         ],
-        "theme": "custom_theme"
+        "theme": "custom_theme",
+        "preventVideoDownload": true
```

```suggestion
        "preventVideoDownload": false
```
This should not be activated by default.